packages/flutter/test/widgets/snapshot_widget_test.dart

```diff
@@ -261,6 +263,39 @@ void main() {
     expect(tester.takeException(), isNull);
     expect(tester.layers.last, isA<PlatformViewLayer>());
   }, skip: kIsWeb); // TODO(jonahwilliams): https://github.com/flutter/flutter/issues/106689
+
+  testWidgets('SnapshotWidget should have same result when enabled', (WidgetTester tester) async {
+    binding.window
+      ..physicalSizeTestValue = const Size(10, 10)
+      ..devicePixelRatioTestValue = 1;
+    addTearDown(() => binding.window
+      ..clearPhysicalSizeTestValue()
+      ..clearDevicePixelRatioTestValue());
+
+    final SnapshotController controller = SnapshotController();
+    await tester.pumpWidget(MaterialApp(
```

Comment:
Add a repaint boundary here and then use the normal method for screenshot testing?

Reply:
done